charms/openfga-k8s/src/charm.py

```diff
@@ -492,6 +505,7 @@ def _create_openfga_store(self, token: str, store_name: str):
     def _list_stores(self, openfga_host: str, headers, continuation_token="") -> list:
         # to list stores we need to issue a GET request to the /stores
         # endpoint
+        # check this
```

Comment:
Check this? :D

Reply:
I take notes to myself in the code and I forgot to remove it xD